.claude/agents/full-dev.md

```
@@ -0,0 +1,615 @@
+1. **DTOs** (`dto/{dto-name}.tsx`)
+   - DTOs(requests and responses) should be typesafe and stricted bound with backend DTOs
+
+```
+// membership/dto/delete-membership-response.tsx
+export interface DeleteMembershipResponse {
+  success: boolean;
+}
+
+```
+
+   - Main data model should be declared as `class` to contain some logic
```

markdownlint-cli2 (0.18.1)

Unordered list indentation
Expected: 0; Actual: 3

(MD007, ul-indent)